apps/wallet-mobile/src/features/Receive/ReceiveNavigator.tsx

```diff
@@ -0,0 +1,61 @@
+import {createStackNavigator} from '@react-navigation/stack'
+import * as React from 'react'
+import {StyleSheet} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+
+import {StatusBar} from '../../components'
+import {defaultStackNavigationOptions, TxHistoryRoutes} from '../../navigation'
+import {useTheme} from '../../theme'
```

Comment:
`useTheme` should be imported from `@yoroi/theme`. Not sure how it works like this as I have an error locally on this import 🤔